<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/6103.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_warning">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0112.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_lamp">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>120</y>
     <width>83</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/6104.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_dankong">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>单控</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_fan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>换气扇开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_warning">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>报警灯开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lamp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>射灯开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rfid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>门禁开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_curtain_on">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>窗帘开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>红外</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_infrared">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>59</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_infrared">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发射</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_curtain_off">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>窗帘关</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_mode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_mode1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>白天模式</string>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_mode2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>夜间模式</string>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_mode3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>安防模式</string>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio_mode4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>离家模式</string>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>传感器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_draw">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>光照度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>温度</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_draw">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>图表</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tv">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>电视机开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dvd">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DVD开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_air">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>空调开</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_ld">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>联动</string>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>条件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld2">
      <item>
       <property name="text">
        <string>&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>值</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld3">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>700</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1000</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>动作</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld5">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>器件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld4">
      <item>
       <property name="text">
        <string>报警灯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>射灯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>门禁</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>换气扇</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="icon_curtain">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/6107.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld1">
      <item>
       <property name="text">
        <string>温度</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>光照度</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>湿度</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
